pypsa/optimization/expressions.py

```diff
@@ -136,7 +139,7 @@ def capex(
         comps: Sequence[str] | str | None = None,
         aggregate_groups: str = "sum",
         aggregate_across_components: bool = False,
-        groupby: list[str] | Callable | None = None,
```

Comment:
I would still allow passing list of strings as this is much more convenient. Otherwise one always has to import the grouper for the function arguments